src/Center.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+
+interface CenterProps{}
+
+export const Center: React.FC<CenterProps> = ({ children }) => {
+    return (
+        <View
+            style = {{
```

Comment:
Inline styles like this are generally a bad practice but we can tackle that in a different PR. For now just know you are wrong and you should feel bad. :)